worldengine/generation.py

```diff
@@ -124,12 +124,17 @@ def initialize_ocean_and_thresholds(world, ocean_level=1.0):
     """
     e = world.elevation['data']
     ocean = fill_ocean(e, ocean_level)
-    hl = find_threshold_f(e, 0.10)
-    ml = find_threshold_f(e, 0.03)
+    pl = find_threshold_f(e, 0.70, ocean=ocean)  # the highest 70% of land are declared plains
+    hl = find_threshold_f(e, 0.35, ocean=ocean)  # the highest 35% are declared hills
+    ml = find_threshold_f(e, 0.10, ocean=ocean)  # the highest 10% are declared low mountains
+    mml = find_threshold_f(e, 0.06, ocean=ocean)  # the highest 10% are declared medium mountains
+    hml = find_threshold_f(e, 0.02, ocean=ocean)  # the highest 10% are declared high mountains
```

Comment:
I added in values for the different mountain-levels. The percentages, however, are merely guessed so far. I could use some better suggestions on those ( @ftomassetti maybe? Also, @esampson seems to know things like these).